src/datasets/features/features.py

```diff
@@ -110,6 +110,8 @@ def _arrow_to_datasets_dtype(arrow_type: pa.DataType) -> str:
         return "string"
     elif pyarrow.types.is_large_string(arrow_type):
         return "large_string"
+    elif pyarrow.types.is_string_view(arrow_type):
+        return "string_view"
```

Comment:
`pa.string_view` exists, so in the complementing function`string_to_arrow("string_view")` will return `pa.StringView` correctly